save-agent/src/nativeTest/kotlin/org/cqfn/save/agent/SaveAgentTest.kt

```diff
@@ -39,6 +39,7 @@ class SaveAgentTest {
                         headersOf(HttpHeaders.ContentType, ContentType.Application.Json.toString())
                     )
                     "/executionData" -> respond("", status = HttpStatusCode.OK)
+                    "/executionLogs" -> respond("", status = HttpStatusCode.OK)
```

Comment:
sleeping for 5 sec can be removed from test, we are blocking on agent call there anyway